pytripgui/main.py

```diff
@@ -1,19 +1,21 @@
 import sys
 import argparse
-import logging
 
 from PyQt5.QtWidgets import QApplication
-
-from pytripgui.view.main_view import MainView
+from pytripgui.main_window_qt_vc.main_window_view import MainWindowQtView
+from pytripgui.main_window_qt_vc.main_window_cont import MainWindowController
 from pytripgui.model.main_model import MainModel
-from pytripgui.controller.main_cont import MainController
 
+
+import logging
```

Comment:
Why are you importing logging module here ? See PEP8 for best practices of organizing imports:
https://www.python.org/dev/peps/pep-0008/#imports

![image](https://user-images.githubusercontent.com/7374733/65906598-21ff0600-e3c3-11e9-850d-814e30d28e03.png)

Reply:
60: logging.basicConfig(level=logging.INFO)

Reply:
I know that `logging` module is needed, but it should be imported first (i.e. just after `import argparse`), as it was before your commit. This corresponds to "1. Standard library imports" from PEP8.

Then `PyQt5` should be imported "2. Related third party imports" and finally local imports should be done ("from pytripgui...". 

Please read the fragment of PEP8 concerning imports